patches/net/minecraft/world/item/AxeItem.java.patch

```diff
@@ -34,17 +58,21 @@
                  if (optional2.isPresent()) {
                      p_308922_.playSound(p_309192_, p_308899_, SoundEvents.AXE_WAX_OFF, SoundSource.BLOCKS, 1.0F, 1.0F);
                      p_308922_.levelEvent(p_309192_, 3004, p_308899_, 0);
-@@ -114,8 +_,19 @@
+@@ -114,8 +_,23 @@
          }
      }
  
-+    @org.jetbrains.annotations.Nullable
 +    public static BlockState getAxeStrippingState(BlockState originalState) {
-+        Block block = STRIPPABLES.get(originalState.getBlock());
-+        return block != null ? block.defaultBlockState().setValue(RotatedPillarBlock.AXIS, originalState.getValue(RotatedPillarBlock.AXIS)) : null;
++        var block = originalState.getBlock().builtInRegistryHolder().getData(net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps.STRIPPABLES);
++        return block != null ? block.stripped().defaultBlockState().setValue(RotatedPillarBlock.AXIS, originalState.getValue(RotatedPillarBlock.AXIS)) : STRIPPABLES.get(originalState.getBlock()).defaultBlockState();
```

Comment:
Calling `defaultBlockState` on the map return is unsafe because the map may not have a value for the block. See the original behavior.